### ISSUE TITLE
when using jsdocker with ssh-agent over ssh it does not work

### ISSUE THREAD

**Issue by despiegk** [Jumpscale/jumpscale_core8#85]
test was
```
jsdocker create -n test --keyname=ovh_install
```
![image](https://cloud.githubusercontent.com/assets/6021844/13545748/0c38f722-e2af-11e5-919d-1f2f60cb7ebb.png)

was in machine over ssh

used 
```
ssh -A root@ovh4 -p 9022
```
to get access

the installtools need to fetch the pub key from the ssh-agent, not from filesystem

**Comment by despiegk** [comment]
use new pubkey=j.do.getSSHKeyFromAgentPub

**Comment by zaibon** [comment]
fix in 48393ebcd2491fd9a234da35c448c2fc07120636
